glossary.md

```diff
@@ -29,6 +29,7 @@ Nếu một từ chưa có trong bảng thuật ngữ dưới đây, các bạn
 | development set                   | tập phát triển                                                 |                                                            |
 | dev set                           | tập phát triển                                                 |                                                            |
 | distribution                      | phân phối                                                      |                                                            |
+| domain adaptation                 | thích ứng miền giá trị                                         |                                                            |
```

Comment:
Từ "domain" trong toán có nghĩa là "miền xác định" nhưng trong trường hợp này không có nghĩa như thế.

"domain" ở đây có nghĩa là "lĩnh vực".

"domain adaptation" chỉ việc một vấn đề đã được giải quyết ở một "source domain" được áp dụng sang một "target domain" mà hai "domain" này có phần nào đó tương tự nhau.

Theo [wikipedia](https://en.wikipedia.org/wiki/Domain_adaptation):
> Domain adaptation[1][2][3] is a field associated with machine learning and transfer learning. This scenario arises when we aim at learning from a source data distribution a well performing model on a different (but related) target data distribution. For instance, one of the tasks of the common spam filtering problem consists in adapting a model from one user (the source distribution) to a new one who receives significantly different emails (the target distribution). Domain adaptation has also been shown to be beneficial for learning unrelated sources.[4] Note that, when more than one source distribution is available the problem is referred to as multi-source domain adaptation.[5]

"thích ứng lĩnh vực" là một cách dịch sát nghĩa hơn, tuy nhiên mình chưa ưng ý với cụm này lắm, cần suy nghĩ thêm.

Reply:
Cảm ơn anh đã làm rõ giúp em. Theo anh nếu dịch thành "sự thích nghi lĩnh vực" hay "sự thích nghi trên lĩnh vực" thì thế nào?

Reply:
"Sự thích nghi lĩnh vực" là từ gần nhất anh có thể nghĩ ra. Em sửa như thế đi rồi dịch vào bài của em. Anh sẽ để một cái note khi dịch từ đó.

Reply:
Vâng, để em làm vậy

Reply:
Ca này khá khó. A cũng chưa nghĩ ra từ nào hay hơn "thích ứng lĩnh vực". Trên wiki cũng mới chỉ có tiếng Anh và tiếng Pháp cho trang này. Nên mình nghĩ cách dịch từ giờ là đẹp.

Reply:
Nếu chỉ để là "thích ứng miền" thì sao nhỉ? Giống như từ "timer" mình chỉ dịch là bộ định thời chứ không dịch bộ định thời gian.

Reply:
Tớ rất thích "thích ứng miền"👍 👍

Reply:
@rootonchair thấy "thích ứng miền" thế nào? Nếu ok thì bạn sửa lại rồi mình approve nhé.

Reply:
@tiepvupsu em cũng thấy phù hợp. Để em sửa lại ở đây và trong bản dịch của em luôn